src/RWAPIIndexer/Resource.php

```diff
@@ -373,8 +373,14 @@ public function index() {
 
     $this->log("Indexing {$this->bundle} entities.\n");
 
+    // Set the number of shards for the index to be created.
+    $shards = $this->options->get('shards') ?? 1;
+
+    // Set the number of replicas for the index to be created.
+    $replicas = $this->options->get('replicas') ?? 1;
```

Comment:
Hmm, this may not actually work if you pass `-R 0`

Reply:
`??` is fine here, `?:` would be the one failing.